Tests/APITesters/ReceiptParserAPITester/ReceiptParserAPITester/AppleReceiptAPI.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AppleReceiptAPI.swift
+//  ReceiptParserAPITester
+//
+//  Created by Nacho Soto on 12/5/22.
+//
+
+import Foundation
+import ReceiptParser
+
+func checkAppleReceiptAPI(_ receipt: AppleReceipt! = nil) {
```

Comment:
Or maybe remove them from the other project?

Reply:
Here's my reasoning:
- Keeping both: we have these APIs exposed as part of `RevenueCat` and `ReceiptParser`, so we need to make sure that they're stable in each
- Keeping them separate: this file is checking the API for `ReceiptParser`. The other file checks `RevenueCat`'s `PurchasesReceiptParser` API. Obviously we share code internally, but the interfaces might not always coincide. `Purchases.ReceiptParser` might have more methods in the future because it's part of the bigger SDK.

Reply:
> Keeping them separate: this file is checking the API for ReceiptParser. The other file checks RevenueCat's PurchasesReceiptParser API. Obviously we share code internally, but the interfaces might not always coincide. Purchases.ReceiptParser might have more methods in the future because it's part of the bigger SDK.

Hmm well, do you have in mind any possible future cases of this scenario? If we do I think it's ok but if not, I would prefer avoiding duplication now for an uncertain scenario.

Reply:
I'm not sure how to do this in a clean way though. The files might be similar but not quite. One has to `import RevenueCat` and the other `ReceiptParser`.

Reply:
Hmm right... I didn't think about that. In that case, I think it's ok to leave as is. Don't want to have a complex setup for the sake of avoiding a bit of duplication.

Reply:
It was a good suggestion though. But yeah not sure it's worth it since we'd need to have some way of dynamically copying contents of the file, etc. etc.